demos/smartwatch/lv_demo_smartwatch_private.h

```diff
@@ -0,0 +1,245 @@
+/**
+ * @file lv_demo_smartwatch_private.h
+ *
+ */
+
+#ifndef LV_DEMO_SMARTWATCH_PRIVATE_H
+#define LV_DEMO_SMARTWATCH_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../lvgl.h"
+
+
+#if LV_USE_DEMO_SMARTWATCH
+
+
+/*********************
+ *      DEFINES
+ *********************/
+
+LV_IMG_DECLARE(img_weather_day_0);          // assets/dy-0.png
```

Comment:
```suggestion
LV_IMG_DECLARE(img_weather_day_0);          /* assets/dy-0.png */
```

And others in the file

Reply:
updated